lib/__tests__/click.test.ts

```diff
@@ -56,86 +65,117 @@ describe("clickedObjectIDsAfterSearch", () => {
     );
   });
 
-  test("Should call sendEvent with proper params", () => {
-    const clickParams = {
-      positions: [1],
-      objectIDs: ["2"],
-      queryID: "testing"
-    };
+  const clickParams = {
+    index: "index1",
+    positions: [1],
+    objectIDs: ["2"],
+    queryID: "testing"
+  };
 
-    analyticsInstance.init(credentials);
-    (analyticsInstance as any).sendEvent = jest.fn();
+  test("should call sendEvent with proper params", () => {
     analyticsInstance.clickedObjectIDsAfterSearch(clickParams);
+    expect((analyticsInstance as any).sendEvent).toHaveBeenCalledWith(
+      "click",
+      clickParams,
+      undefined
+    );
+  });
 
+  test("should call sendEvents with additional params if provided", () => {
+    analyticsInstance.clickedObjectIDsAfterSearch(
+      clickParams,
+      additionalParameters
+    );
     expect((analyticsInstance as any).sendEvent).toHaveBeenCalledWith(
```

Comment:
I guess it's consistent, but do you know why they have to be casted?

Reply:
In v1 `AlgoliaAnalytics.sendEvent()` is a protected method. I could replace this with `// @ts-expect-error` as I did for other assertions in those tests if you prefer.

Reply:
either way is fine I think